homeassistant/components/vizio/config_flow.py

```diff
@@ -334,6 +348,10 @@ async def async_step_zeroconf(
 
         # Check if new config entry matches any existing config entries and abort if so
         for entry in self.hass.config_entries.async_entries(DOMAIN):
+            # If source is ignore bypass host check and continue through loop
+            if entry.source == SOURCE_IGNORE:
+                continue
+
             if _host_is_same(entry.data[CONF_HOST], discovery_info[CONF_HOST]):
                 return self.async_abort(reason="already_setup")
```

Comment:
Ignored entries should be treated as already set up. Now you're ignoring the ignored, causing it to show up as a discovery?

Reply:
The purpose of this is to not do a host check when iterating over an ignored config entry because the `host` key won't be a part of the entry

Reply:
Are integrations expected to handle the abort for an ignored entry? If so I can submit a separate PR for this

Reply:
Lol scratch my previous comment, I see the issue. Will submit a separate PR